code/notificationbuilder/src/test/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/ManualCarouselNotificationBuilderTest.kt

```diff
@@ -127,11 +132,44 @@ class ManualCarouselNotificationBuilderTest {
         }
     }
 
+    @Test
+    fun `construct returns ManualCarouselNotificationBuilder if download image count is greater than 3`() {
+        every { cacheImages(any()) } answers { 5 }
+        ManualCarouselNotificationBuilder.construct(
+            context,
+            pushTemplate,
+            trackerActivityClass,
+            broadcastReceiverClass
+        )
+        verify(exactly = 0) {
+            BasicNotificationBuilder.fallbackToBasicNotification(
+                context,
+                trackerActivityClass,
+                broadcastReceiverClass,
+                pushTemplate.data
+            )
+        }
+    }
+
+    @Test
+    fun `construct returns ManualCarouselNotificationBuilder if downloadCarouselItems failed to download some of the images`() {
```

Comment:
should a call to ManualCarouselNotificationBuilder.construct() be made in this test case? the verification step could then verify that BasicNotificationBuilder.fallbackToBasicNotification() is not called.

Reply:
updated the name of the test @rymorale